src/is/arithmetic.js

```diff
@@ -20,7 +20,19 @@ export function isOdd(value) {
     return value % 2 !== 0;
 }
 
+/**
+ * Determines whether a given value is an integer
+ * @param {*} value the value to check
+ * @returns {Boolean} `true` if the value is an integer; otherwise, `false`
+ */
+export function isInteger(value) {
+    if (!isNumber(value)) return false;
+    return value % 1 === 0;
```

Comment:
Less performant than checking if the rounded value is equal to the initial value